charts/producer-app-cleanup-job/templates/job.yaml

```diff
@@ -15,19 +15,19 @@ spec:
   ttlSecondsAfterFinished: 30
   template:
     metadata:
-    {{- if or .Values.awsRole .Values.annotations }}
```

Comment:
annotations are not used anymore. Either we reuse them from the values or we add them to the job metadata (see above, labels are there). Any opinion @torbsto ?

Reply:
I think we should just add them for consistency